integration-tests/spring-data-jpa/src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
+quarkus.package.type=fast-jar
```

Comment:
I am wondering whether uber-jar would be a better choice here, considering that fast-jar will soon become the default one? And perhaps be renamed to just 'jar' or something?